project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
```

Comment:
Good point; wondering if that should be part of *this* document, or if we should create a checklist, like we have for "major" releases; https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md

But yes, other projects depend on these releases (boot2docker, toolbox)